### PR TITLE
[NEW] Include subscriptions.read new REST API

### PR DESCRIPTION
**Pull request by rafaelks**

### CODE REVIEW

File: developer-guides/rest-api/subscriptions/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Subscriptions
```
Comment (MartinSchoeler) [inline developer-guides/rest-api/subscriptions/README.md]: @rafaelks Could you write the description of this api? like in [this file](https://github.com/RocketChat/docs/blob/master/developer-guides/rest-api/miscellaneous/README.md) Thanks!

Comment (rafaelks) [reply developer-guides/rest-api/subscriptions/README.md]: This is not an actual API... can I remove the README file just to maintain the `subscriptions` folder?

Comment (MartinSchoeler) [reply developer-guides/rest-api/subscriptions/README.md]: @rafaelks if you do that, when someone clicks on the folder, they will be redirected to the home page, do you plan to add more files to that folder? Maybe we can just put a list of what is inside of the folder

Comment (rafaelks) [reply developer-guides/rest-api/subscriptions/README.md]: Done! 👍

**Review by MartinSchoeler — APPROVED** [review]

**Comment by rafaelks** [comment]
@MartinSchoeler Let's revert this one for now, sorry... the API is not merged yet.

**Comment by MartinSchoeler** [comment]
@rafaelks i will just remove it from the table of contents for now, people wont be able to navigate to there

**Comment by rodrigok** [comment]
@rafaelks @MartinSchoeler the API was merged, how are you controlling the versioning of the API related to the chat version?

**Comment by philipbrito** [comment]
@rafaelks @MartinSchoeler The [URL](https://rocket.chat/docs/developer-guides/rest-api/subscriptions/) for this documentation works, but it isn't showing on the REST API section (on the left side)

Update: https://rocket.chat/docs/developer-guides/rest-api/subscriptions/read/ shows an empty content.

**Comment by rafaelks** [comment]
@MartinSchoeler Can we fix the docs for this please? Not sure what's happening.

**Pull request merged**